components/InfoCard.tsx

```diff
@@ -0,0 +1,36 @@
+import { Card, CardActionArea, CardContent } from '@mui/material'
+import Router from 'next/router'
+import React from 'react'
+
+function InfoCard({ prospect, member }) {
+  function goToProspectProfile() {
+    Router.push('/prospects/profile/' + prospect.id)
+  }
+
+  return (
+    <>
+      { prospect && 
+        <CardActionArea onClick={goToProspectProfile}>
+          <Card  sx={{ width: 'auto', height: 'auto', m: 1}} elevation={5}>
+            <CardContent id={'prospect-' + prospect.id}>
+              <h2>{prospect.name}</h2>
+              <p>{prospect.mail}</p>
+              <p>{prospect.phone}</p>
+            </CardContent>
+          </Card>
+        </CardActionArea>
+      }
+      { member &&
+        <Card  sx={{ width: 'auto', height: 'auto', m: 1}} elevation={5}>
+          <CardContent id={'member-' + member.id}>
+            <h2>{member.name}</h2>
+            <p>{member.mail}</p>
+            <p>{member.role}</p>
+          </CardContent>
```

Comment:
This is still repetitive code, but in the same file. We need to get rid completely of the repetitive code